functions/import-d365aaduser.ps1

```diff
@@ -157,15 +178,37 @@ function Import-D365AadUser {
 
     $azureAdUsers = New-Object -TypeName "System.Collections.ArrayList"
 
-    if ( $PSCmdlet.ParameterSetName -eq "GroupImport") {
+    if (( $PSCmdlet.ParameterSetName -eq "GroupNameImport") -or ($PSCmdlet.ParameterSetName -eq "GroupIdImport")) 
+    {
 
-        $group = Get-AzureADGroup -SearchString $AadGroupName
+        if ($PSCmdlet.ParameterSetName -eq 'GroupIdImport')
+        {
+            Write-PSFMessage -Level Verbose -Message "Search AadGroup by its ID : $AadGroupId"
+            $group = Get-AzureADGroup -ObjectId $AadGroupId
+        }
+        else
+        {
+            if ($ForceExactAadGroupName -eq $true)
+            {
+                Write-PSFMessage -Level Verbose -Message "Search AadGroup by its exaclty name : $AadGroupName"
+                $group = Get-AzureADGroup -Filter "DisplayName eq '$AadGroupName'"
+            }
+            else
+            {
+                Write-PSFMessage -Level Verbose -Message "Search AadGroup by searching with its name : $AadGroupName"
+                $group = Get-AzureADGroup -SearchString $AadGroupName
+            }
+        }
 
         if ($null -eq $group) {
             Write-PSFMessage -Level Host -Message "Unable to find the specified group in the AAD. Please ensure the group exists and that you have enough permissions to access it."
             Stop-PSFFunction -Message "Stopping because of errors"
             return
         }
+        else
+        {
+            Write-PSFMessage -Level Host -Message "Processing Azure AD user Group ""$($group[0].DisplayName)"""
```

Comment:
Write-PSFMessage -Level Host -Message "Processing Azure AD user Group $($group[0].DisplayName)"

Reply:
Mötz, it seems that working with my "" 
![image](https://user-images.githubusercontent.com/40988263/46458423-05cb8780-c7b6-11e8-8d8d-8f19b2a6502a.png)


but i change this line to apply our recommdation and create a new pull request

Reply:
hum the code formatting is better with your solution, see directly that a variable is added to a string content